packages/backend/src/api/request.ts

```diff
@@ -1,3 +1,4 @@
+import { ClerkAPIResponseError } from '@clerk/shared';
```

Comment:
@nikosdouvlis Do we foresee any building issues with this import from shared? Should we wait for the @clerk/shared architecture as part of #1898 ?

Reply:
@Nikpolik could you wait until the https://github.com/clerkinc/javascript/pull/1898 is merged and use the subpath import? Not using the subpath import would cause the backend package to fail in node runtimes that do not have react as peer dependency and bundling tools like vite.

Reply:
👍

Reply:
@Nikpolik you can proceed with this PR